uitestcore/finder.py

```diff
@@ -25,7 +25,10 @@ def elements(self, page_element):
         :param page_element: PageElement instance representing the element
         :return: list of matching WebElements
         """
-        return self.driver.find_elements(page_element.locator_type, page_element.locator_value)
+        self.logger.info(f"Looking for elements matching {page_element}")
```

Comment:
Is INFO the right level, how about DEBUG?

Reply:
I tried it with debug first but there's so much other stuff that is logged out that it gets lost in the noise. Do you think it needs to be logged as debug if it would only be used for debugging?

Reply:
No, let's have it as info for now. It can always change later